vars/pluginUtils.groovy

```diff
@@ -0,0 +1,88 @@
+def runPluginTests(plugins, envName, markerName, slackChannel, slackUrl) {
+    def lockName = "${env.JOB_NAME}-${envName}"
+    def pluginNames = plugins.keySet()
+
+    if (markerName != "") {
+        lockName = lockName + "-${markerName}"
```

Comment:
I think locking only on JOB_NAME/ENV NAME is adequate.

The purpose of the lock is to ensure that two jobs do not run against the same service in the same env at once. If we add 'markerName' to the end I would think we'd open up the possibility of 2 test jobs w/ different markers running against the same app at the same time?